db/migrate/20160101021129_populate_languages.rb

```diff
@@ -0,0 +1,10 @@
+class PopulateLanguages < ActiveRecord::Migration
+  def up
+    Language.create({id:1,name:'English',code:'EN'})
+    Language.create({id:2,name:'Español',code:'ES'})
```

Comment:
Why's this being done as a migration rather than in `seeds.rb`? I usually try to avoid putting seed data in the migrations, it can mess with the test suite which cleans out the database between test runs. If it were in the `seeds.rb` file that data could be required for the specs that rely on its presence.